CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
```

Comment:
Would you be okay with adding this bullet point here:
* "Dismissing or attacking inclusion-oriented requests"
?

---

I also really love [forem](https://dev.to/code-of-conduct)'s CoC, which includes an additional very explicit subsection in this part, which reads:

> We prioritize marginalized people's safety over privileged people's comfort. We will not act on complaints regarding:
> 
> * ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
> * Reasonable communication of boundaries, such as 'leave me alone,' 'go away,' or 'I’m not discussing this with you.'
> * Someone’s refusal to explain or debate social justice concepts
> * Criticisms of ableist, racist, sexist, homophobic, transphobic, or otherwise oppressive behavior or assumptions
> 
> In community communication, we do not appreciate any form of aggression (passive aggression included) — please don't post stuff if you can't be kind and empathetic. Be proactive about owning your mistakes and apologizing if you have a feeling or know that you impacted someone in a negative way.
> 

How would you feel about adding it here as well?

Reply:
> How would you feel about adding it here as well?

Looks great. Right beloe?